CHANGELOG.md

```diff
@@ -6,23 +6,37 @@ Changes by Version
 
 #### Backend Changes
 
-- Remove cassandra SASI indices [#1328](https://github.com/jaegertracing/jaeger/pull/1328)
+##### Breaking Changes
+
+- Remove cassandra SASI indices ([#1328](https://github.com/jaegertracing/jaeger/pull/1328), [@black-adder](https://github.com/black-adder))
```

Comment:
I have moved this to breaking changes even it is not. It better catches attention.